### DISCUSSION TITLE
chromium/chromium broken for search

### DISCUSSION THREAD

**Discussion by cachius** [community/community#32445] [Other Feature Feedback, Questions, & Ideas] [answered]
[chromium/chromium](https://github.com/chromium/chromium) seems to be in a somewhat broken state, at least for search:

### Search for /.*/
[brave/brave-core](https://github.com/brave/brave-core/search?q=%2F.*%2F) 13.610 results
[chromium/chromium](https://github.com/chromium/chromium/search?q=%2F.*%2F) 38 results
[torvalds/linux](https://github.com/torvalds/linux/search?q=%2F.*%2F) 78,014 results

At this point it ocurred to me that searching for `/.*/` should match all files in a repo and probably does.
I tested this on some small repos and it seems to hold:

[stencila/test](https://github.com/stencila/test/search?q=%2F.*%2F) 3 results  of 3 files total
[rtyley/small-test-repo](https://github.com/rtyley/small-test-repo/search?q=%2F.*%2F) 1 result of 1 file total

### Search for RegisterURLSchemeAsSupportingFetchAPI
[brave/brave-core](https://github.com/brave/brave-core/search?q=RegisterURLSchemeAsSupportingFetchAPI) 1 result
[chromium/chromium](https://github.com/chromium/chromium/search?q=RegisterURLSchemeAsSupportingFetchAPI) 0 results

### A closer look
... on the chromium results reveals: → all _Last indexed on 9 May_!

So search indexing seems to have stopped then
and only 38 files at all seem to have remained in the index.

### Questions that come to mind are
- Can this be fixed?
- Can turning search off set intentionally by the repo owners?
- Is only search broken or does it affect other areas as well?
- Is only chromium/chromium affected?
  - How to tell when a repository is affected?

Please have a look into the issue.
Kind regards!

**Comment by cachius** [comment]
@colinwm Could you maybe look into or say something about this, please?

**Comment by jfullstackdev** [comment]
you can ask questions there at the Issues tab regarding that, others with the same interest might help or the authors themselves

**Reply by cachius** [reply]
It is only a mirror, as per About section:
>The official GitHub mirror of the Chromium source

So there [is no issues tab](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/disabling-issues) and I'm not sure how to reach out.

Besides, I couldn't find any settings related to search or indexing under [Manage repository settings](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features).
That's why I guess the behavior is unintended or a bug.

@github-staff Please verify, maybe with development. For recap:
https://github.com/chromium/chromium/search?q=test finds 22 results vs
https://github.com/brave/brave-core/search?q=test finds 1861.

**Comment by cachius** [comment, marked as answer]
As workaround (and reference whether search works as intended for @github-staff) one can <br>
**Use the [Chromium Code Search](https://source.chromium.org/chromium) hosted by Google**:

https://source.chromium.org/search?q=test&ss=chromium 76,610 results
https://source.chromium.org/search?q=RegisterURLSchemeAsSupportingFetchAPI&ss=chromium 8 results

This seems a hard problem as well as often needed if Google provides a dedicated solution for it.

I'd not expect GitHub to match their solution but the current state is misleading,
showing merely a few search results giving the impression that those are all.
